cmd/snap/main.go

```diff
@@ -281,7 +281,9 @@ func main() {
 
 	// magic \o/
 	snapApp := filepath.Base(os.Args[0])
-	if osutil.IsSymlink(filepath.Join(dirs.SnapBinariesDir, snapApp)) {
+	switch {
+	case osutil.IsSymlink(filepath.Join(dirs.SnapBinariesDir, snapApp)):
```

Comment:
so... a snap `command-not-found` would override the builtin c-n-f.

I ... this might be fine.